cmd/rofl/build.go

```diff
@@ -71,14 +71,10 @@ var (
 			case buildMode == buildModeAuto && !offline:
 				ctx := context.Background()
 				conn, err := connection.Connect(ctx, npa.Network)
-				if err != nil {
-					break // Autodetection failed.
-				}
+				cobra.CheckErr(err)
 
 				params, err := conn.Consensus().Registry().ConsensusParameters(ctx, consensus.HeightLatest)
-				if err != nil {
-					break // Autodetection failed.
-				}
+				cobra.CheckErr(err)
```

Comment:
This should produce a more verbose error message saying that autodetection failed, and to use `--mode` manually.